<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile_Window_Dialog</class>
 <widget class="QDialog" name="Profile_Window_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #73C2FB; /* Ice Blue*/
    font-family: 'Segoe UI', sans-serif;
}</string>
  </property>
  <widget class="QLabel" name="Photo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Name_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="User_id_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>SJSU ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="Email_Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Email id</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="CourseName_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Course Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Enrollment_Number_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Enrollment No.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Department">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Department :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Admission_YearLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Admission Year :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HOME_Label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="Account_Status_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Account Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Contact_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FAFAD2;
    font-size: 14px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Mobile Phone</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>20</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Photo_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Library/Mobile Documents/com~apple~CloudDocs/Shreya/SJSU/DATA201_RON/Project/FinalProject/Images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
